api/util/apps.go

```diff
@@ -41,6 +44,19 @@ func EnvVarsFromMap(env map[string]string) apps.EnvVars {
 	return vars
 }
 
+func EnvVarToString(envs apps.EnvVars) string {
+	if envs == nil {
+		return "nil"
+	}
+
+	var s strings.Builder
+	for _, env := range envs {
+		s.WriteString(fmt.Sprintf("%v=%v;", env.Name, env.Value))
```

Comment:
A bit of a nit: If there is just one env it will be `key=val;`, which looks a bit off. I guess we could construct a `[]string` with just `key=val` for each env and then do a return `strings.Join([]string, ";")` at the end.